metricbeat/module/kubernetes/_meta/config.yml

```diff
@@ -21,3 +21,10 @@
     - state_container
   period: 10s
   hosts: ["kube-state-metrics:8080"]
+
+# Kubernetes events
+- module: kubernetes
+  metricsets:
+    - event
```

Comment:
I remember @exekias added somewhere that all `state_*` prefixed metricset must be installed once in contrast to the other ones. If this is installed global (my assumption) should this also have a `state_` prefix?

NOTE: In general I think we need to find a better solution for this in the long run but in the short term we should try to be consistent.